docs/cli.md

```diff
@@ -0,0 +1,38 @@
+# Promscale command line doc
+
+This document gives you information about the configuration flags supported by promscale.
+You can also find information on flags with `promscale_<version> -help` 
+
+## General flags
+| Flag | Type | Default | Description |
+|------|:-----:|:-------:|:-----------:|
+| install-timescaledb | boolean | true | Install or update TimescaleDB extension. |
+| leader-election-pg-advisory-lock-id | integer | 0 (disabled) | Leader-election based high-availability. It is based on PostgreSQL advisory lock and requires a unique advisory lock ID per high-availability group. Only a single connector in each high-availability group will write data at one time. A value of 0 disables leader election. |
+| leader-election-pg-advisory-lock-prometheus-timeout | slack/duration | -1 | Prometheus timeout duration for leader-election high-availability. The connector will resign if the associated Prometheus instance does not respond within the given timeout. This value should be a low multiple of the Prometheus scrape interval, big enough to prevent random flips. |
```

Comment:
```suggestion
| leader-election-pg-advisory-lock-prometheus-timeout | slack/duration | -1 | Prometheus timeout duration for leader-election high-availability. The connector will resign if the associated Prometheus instance does not send any data within the given timeout. This value should be a low multiple of the Prometheus scrape interval, big enough to prevent random flips. |
```